.github/workflows/test-installers.yml

```diff
@@ -0,0 +1,26 @@
+name: Test Installers
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: 'Version of the release. e.g., swan-lake-beta1'
+        required: true
+      taggedVersion:
+        description: 'Tagged version of the release. e.g., swan-lake-beta1-rc1, swan-lake-beta1'
```

Comment:
@NipunaMadhushan Shall we only expect only the pre release part as the input (eg: "-rc1" or "rc1"). Then we can construct entire thing with "releaseVersion" input

Reply:
@keizer619 I will address the suggestions.